src/view-components/view-utils.test.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import { render, waitForElement, cleanup } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+import { getView } from "./view-utils";
+import {
+  widgetDefinitions as coreWidgetDefinitions,
+  dashboardDefinitions as coreDashboardDefinitions,
+  tabbedViewDefinitions as coreTabbedViewDefinitions
+} from "./core-views";
+
+declare var System;
```

Comment:
mock System  globally in the jest config

Reply:
done